File: App/Clients/ng-microservices-demo-client/src/app/infrastructure/rest-clients/policies/models/responses/policy-response.ts
import { AutoMap } from "@automapper/classes";

export class PolicyResponse {
  @AutoMap()
  public number: string;
  @AutoMap()
  public dateFrom: Date;
  @AutoMap()
  public dateTo: Date;
  @AutoMap()
  public policyHolder: string;
  @AutoMap()
  public totalPremium: number;
  @AutoMap()
  public productCode: string;
  @AutoMap()
  public accountNumber: string;
  @AutoMap()
  public covers: string[];

  constructor(
    number: string,
    dateFrom: Date,
    dateTo: Date,
    policyHolder: string,
    totalPremium: number,
    productCode: string,
    accountNumber: string,
    covers: string[]
  ) {
    this.number = number;
    this.dateFrom = dateFrom;
    this.dateTo = dateTo;
    this.policyHolder = policyHolder;
    this.totalPremium = totalPremium;
    this.productCode = productCode;
    this.accountNumber = accountNumber;
    this.covers = covers;
  }
}

File: App/Services/Microservices.Demo.Policies.Service/Framework/Rest/Models/Responses/PolicyResponse.cs

using Microservices.Demo.Policies.Service.Framework.Rest.Models.Dtos;

namespace Microservices.Demo.Policies.Service.Framework.Rest.Models.Responses
{
    public class PolicyResponse
    {
        public string Id { get; set; }
        public string Number { get; set; }
        public DateTime DateFrom { get; set; }
        public DateTime DateTo { get; set; }
        //public string PolicyHolder { get; set; }
        public decimal TotalPremium { get; set; }
        public string ProductCode { get; set; }
        public string AccountNumber { get; set; }

        public List<string> Covers { get; set; }

        public PolicyHolderDto PolicyHolder { get; set; }
    }
}
